websites/V/Vizjer/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "334411435633541121",
+		"name": "meffiu"
+	},
+	"service": "Vizjer.pl",
+	"description": {
+		"en": "Polish site with free movies and series without registration.",
+		"pl": "Polska strona z darmowymi filmami i serialami bez rejestracji."
+	},
+	"url": "vizjer.pl",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/K8NJGp3.png",
+	"thumbnail": "https://i.imgur.com/D1i1HI1.png",
+	"color": "#BABABA",
+	"category": "videos",
+	"tags": [
+		"video",
+		"media",
+		"movies",
+		"series",
+		"tv"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*"
```

Comment:
Please change this to something more specific if possible.

Reply:
i just copied it from another my presence that uses iframe, im not sure what more specific i should change to

Reply:
For example, let's say the iframes that you need to fetch data from are located in `example.com` and `video.foo.com`, you could do a regex like: `example[.]com|video[.]foo[.]com`

Reply:
only for domains?